server/cli/flags.go

```diff
@@ -105,6 +105,10 @@ var flagUsage = map[string]string{
         ranges. The scan is slowed as necessary to approximately achieve this
         duration.
 `,
+	"scan-max-idle-time": `
```

Comment:
want to rip out the `ScanInterval` command line parameter? We didn't want to expose it but for its use as a quickfix, which now is obsolete.